src/connectionController.ts

```diff
@@ -77,6 +79,35 @@ interface ConnectionSecretsInfo {
 type StoreConnectionInfoWithConnectionOptions = StoreConnectionInfo &
   Required<Pick<StoreConnectionInfo, 'connectionOptions'>>;
 
+export class KeytarModuleUnavailableError extends Error {
+  name = 'KeytarModuleUnavailableError';
```

Comment:
Do you know if we need to set a name explicitly here? I think `err.constructor.name` should work based on the class name. And maybe I missing it somewhere, but seem you don't read it anywhere and only check `instanceof KeytarModuleUnavailableError`.

Reply:
yea that's right. I thought I would use it but did not. Will remove it, thank you :D

Reply:
Fwiw, not strong feelings either way I think it’s a good practice to set a `name` property for `Error` subclasses and use that for instance checking rather than `instanceof` or `err.constructor.name` (because it works across contexts and `err.name` is [already standard built-in JS](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/name))